### PR TITLE
BugFix themes link

### PR DESCRIPTION
**Pull request by cavasinf**
## Description
Fix themes link in doc

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] I updated the documentation (see [here](https://github.com/kevinpapst/TablerBundle/tree/master/docs))
- [x] I agree that this code will be published under the [MIT license](https://github.com/kevinpapst/TablerBundle/blob/master/LICENSE)

### CODE REVIEW

File: docs/form_theme.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Form theme
 
 This bundle does ship three form-themes:
-- [templates/layout/form-theme.html.twig](templates/layout/form-theme.html.twig)
-- [templates/layout/form-theme-horizontal.html.twig](templates/layout/form-theme-horizontal.html.twig)
-- [templates/layout/form-theme-security.html.twig](templates/layout/form-theme-security.html.twig)
+- [templates/layout/form-theme.html.twig](/templates/layout/form-theme.html.twig)
```
Comment (kevinpapst) [inline docs/form_theme.md:4]:
```suggestion
- [templates/layout/form-theme.html.twig](https://github.com/kevinpapst/TablerBundle/blob/main/templates/layout/form-theme.html.twig)
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
I'd say the only way to make it universally working is to use absolut links?!

File: docs/form_theme.md (hunk 1 above)
Comment (cavasinf) [reply docs/form_theme.md:4]: I don't think so.

If in `main` branch:
- `/xxxx` means `https://github.com/kevinpapst/TablerBundle/blob/main/xxxx`
- `xxxx` means `https://github.com/kevinpapst/TablerBundle/blob/main/{{pathToCurrentFile}}/xxxx`

If we especially prefix links with `https://github.com/kevinpapst/TablerBundle/blob/main`,
that means every forks of the bundle with any branches will redirect to `main` from here. 
It does not matter which depot or branch you are in.

Comment (kevinpapst) [reply docs/form_theme.md:4]: I do not care about forks... and who is reading branch docs? 

Anyway:

> * `/xxxx` means `https://github.com/kevinpapst/TablerBundle/blob/main/xxxx`

Is that true? Is Github dynamically replacing these links?
I learned something new today and your solution looks good then!

**Review by kevinpapst — APPROVED** [review]

File: docs/form_theme.md (hunk 1 above)
Comment (cavasinf) [reply docs/form_theme.md:4]: > I do not care about forks... 

I'm in that case for FOSUserBundle, forked it to not have swift mailer dependency but still have doc updated.

> and who is reading branch docs?

Some PR will update docs, new component/macro/..
In PR user side, his links should point to `main` branch ?
Kinda weird to me.

> Is that true? Is Github dynamically replacing these links?

Yeah, you can check on main branch now it's merged 😉 
https://github.com/kevinpapst/TablerBundle/blob/main/docs/form_theme.md

Comment (kevinpapst) [reply docs/form_theme.md:4]: That was more of a rhetorical question, I tried your changed file directly in the PR via "View file" and it worked 😁 so I knew you were right. I just didn't know that or just forgot about that feature... but it makes writing docs much easier, which is really good to know!

**Pull request merged**